src/events/actions/updateWebsiteActions/generateVoxxrinJson.ts

```diff
@@ -0,0 +1,203 @@
+        theming: {
+            colors: {
+                primaryHex: string
+                primaryContrastHex: string
+                secondaryHex: string
+                secondaryContrastHex: string
+                tertiaryHex: string
+                tertiaryContrastHex: string
+            }
+        }
+        supportedTalkLanguages: { id: string; themeColor: string; label: string }[]
+        rooms: { id: string; title: string }[]
+        talkTracks: { id: string; title: string; themeColor: string }[]
+        talkFormats: { id: string; title: string; duration: string; themeColor: string }[]
+        features: {
+            favoritesEnabled: boolean
+            roomsDisplayed: boolean
+            remindMeOnceVideosAreAvailableEnabled: boolean
+            showInfosTab: true
+            showRoomCapacityIndicator: false
+            hideLanguages: []
+            ratings: {
+                scale: {
+                    enabled: boolean
+                }
+                bingo: {
+                    enabled: boolean
+                }
+                'free-text': {
+                    enabled: boolean
+                }
+            }
+        }
+        sponsors: {
+            type: string
+            typeBackgroundColor: string
+            typeFontColor: string
+            sponsorships: {
+                name: string
+                logoUrl: string
+                href: string
+            }[]
+        }[]
+        sessions: {
+            [key: string]: any
+        }
+        speakers: {
+            [key: string]: any
+        }
+    } = {
+        title: event.name,
+        headingTitle: event.name,
+        timezone: eventTimezone,
+        start: event.dates.start?.toISOString(),
+        end: event.dates.end?.toISOString(),
+        days: days.map((day) => ({
+            id: day.start.toISO() || Math.random().toString(),
+            localDate: day.start.toLocaleString(),
+        })),
+        logoUrl: logoUrl,
+        backgroundUrl: backgroundUrl,
+        theming: {
+            colors: {
+                primaryHex: event.color || '#F78125',
+                primaryContrastHex: hexContrast(event.color || '#F78125'),
+                secondaryHex: event.colorSecondary || '#3880FF',
+                secondaryContrastHex: hexContrast(event.colorSecondary || '#3880FF'),
+                tertiaryHex: event.colorBackground || '#202020',
+                tertiaryContrastHex: hexContrast(event.colorBackground || '#202020'),
+            },
+        },
+        supportedTalkLanguages: supportedTalkLanguages,
+        rooms: event.tracks.map((track) => ({ id: track.id, title: track.name })),
+        talkTracks: event.categories.map((category) => ({
+            id: category.id,
+            title: category.name,
+            themeColor: category.color || '#000000',
+        })),
+        talkFormats: event.formats.map((format) => ({
+            id: format.id,
+            title: format.name,
```

Comment:
@HugoGresse if possible, it would be nice to not integrate talk format name into the `title` because I'm appending it in the UI (so, currently, it's displayed twice)

<img width="465" alt="Cursor_and_Voxxrin" src="https://github.com/HugoGresse/OpenPlanner/assets/603815/a6626959-cdac-4624-9459-f765c1f9889f">

Reply:
This is a per event settings: each organizers can put what they need to the format name, for SunnyTech, we've added the format duration in plain text

Reply:
@HugoGresse would this setting have an impact elsewhere ? (I mean, if sunnytech would remove duration from their format title to comply with Voxxrin expectations, would it have an undesired side effect elsewhere on their end (for instance, on their website))

On my side, I might make the "auto append duration after format title" configurable

Reply:
For now, the format is not visible at all, so no side effect, it's more for the "OpenPlanner" part, which could also be solved in another PR. 
For Voxxrin, maybe manaing the format name as a regular format, and let the organizer fix their error if needed would be a good option no?

Reply:
@HugoGresse Would it be possible for you to tell Voxxrin if duration is included in format title (or not), based on your event settings, through a voxxrin feature flag ?

I was thinking about : 
```
{
...
  "features": {
    ...
    "formattings": {
      "talkFormatTitle": "with-duration" // can also be set to "without-duration" based on openplanner setting
    }
    ...
  }
...
}
```

WDYT ?

Reply:
I don't think it would make sense. 
What should be done: 
- if duration is needed for organization purpose, just update the UI in OpenPlanner
- let the organizer remove the duration from the format which should solve this issue without more dev

Reply:
Done!